http/org.wso2.carbon.transport.http.netty/src/test/java/org/wso2/carbon/transport/http/netty/https/CipherSuitesTest.java

```diff
@@ -0,0 +1,188 @@
+        listenerConfiguration.setCertPass(password);
+        listenerConfiguration.setScheme(scheme);
+        listenerConfiguration.setParameters(serverParams);
+
+        ServerConnector serverConnector = factory
+                .createServerConnector(ServerBootstrapConfiguration.getInstance(), listenerConfiguration);
+        ServerConnectorFuture future = serverConnector.start();
+        future.setHttpConnectorListener(new EchoMessageListener());
+        future.sync();
+
+        HttpWsConnectorFactory connectorFactory = new HttpWsConnectorFactoryImpl();
+        httpClientConnector = connectorFactory
+                .createHttpClientConnector(HTTPConnectorUtil.getTransportProperties(transportsConfiguration),
+                        HTTPConnectorUtil.getSenderConfiguration(transportsConfiguration, Constants.HTTPS_SCHEME));
+
+        testCiphersuites(hasException, serverPort);
+        serverConnector.stop();
+        httpClientConnector.close();
+    }
+
+    public void testCiphersuites(boolean hasException, int serverPort) {
+        try {
+            ByteBuffer byteBuffer = ByteBuffer.wrap(testValue.getBytes(Charset.forName("UTF-8")));
+            HTTPCarbonMessage msg = new HTTPCarbonMessage(
+                    new DefaultHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.GET, ""));
+            msg.setProperty("PORT", serverPort);
+            msg.setProperty("PROTOCOL", scheme);
+            msg.setProperty("HOST", TestUtil.TEST_HOST);
+            msg.setProperty("HTTP_METHOD", "GET");
+            msg.addMessageBody(byteBuffer);
+            msg.setEndOfMsgAdded(true);
+
+            CountDownLatch latch = new CountDownLatch(1);
+            SSLConnectorListener listener = new SSLConnectorListener(latch);
+            HttpResponseFuture responseFuture = httpClientConnector.send(msg);
+            responseFuture.setHttpConnectorListener(listener);
+            latch.await(3, TimeUnit.SECONDS);
+            HTTPCarbonMessage response = listener.getHttpResponseMessage();
+
+            if (hasException) {
+                assertNotNull(listener.getThrowables());
+                boolean hasSSLException = false;
+                for (Throwable t : listener.getThrowables()) {
+                    if (t.getMessage() != null && t.getMessage().contains("handshake_failure")) {
+                        hasSSLException = true;
+                        break;
+                    }
+                }
+                assertTrue(hasSSLException);
+            } else {
+                assertNotNull(response);
+                String result = new BufferedReader(
+                        new InputStreamReader(new HttpMessageDataStreamer(response).getInputStream())).lines()
+                        .collect(Collectors.joining("\n"));
+                assertEquals(testValue, result);
+            }
+        } catch (Exception e) {
+            TestUtil.handleException("Exception occurred while running testCiphersuites", e);
+        }
+
```

Comment:
This white spaces are not needed. I suppose.

Reply:
Removed